toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/GeoViewScope.kt

```diff
@@ -541,3 +525,22 @@ private fun calloutPath(
         close()
     }
 }
+
+/**
+ * This function is used to wait for the GeoView to be ready to return positive values
+ * for operations like locationToScreen. We determine that by waiting for the first drawStatus
+ * message when the map/scene is rendered on the GeoView.
+ * For the MapView/SceneView's content parameter like the Callout we don't want to start drawing
+ * the Callout until the GeoView is ready.
+ *
+ * @since 200.5.0
+ */
+@Composable
+public fun GeoView.rememberIsReady(): State<Boolean> {
```

Comment:
I think we should make it internal for now, unless we know of a public use case:
```suggestion
internal fun GeoView.rememberIsReady(): State<Boolean> {
```